src/vm/arm/stubs.cpp

```diff
@@ -2522,6 +2522,12 @@ void UMEntryThunkCode::Encode(BYTE* pTargetCode, void* pvSecretParam)
     FlushInstructionCache(GetCurrentProcess(),&m_code,sizeof(m_code));
 }
 
+void UMEntryThunkCode::Poison()
+{
+    m_code[0] = 0xbebe;
+    FlushInstructionCache(GetCurrentProcess(),&m_code,sizeof(m_code));
```

Comment:
I do not think `FlushInstructionCache` is needed here - it is not free. This aid is not 100% reliable because of it depends on the GC timings, making it a bit less reliable by skipping the `FlushInstructionCache` here should be ok.